aci/resource_aci_infrarsdomp.go

```diff
@@ -23,23 +23,26 @@ func resourceAciInfraRsDomP() *schema.Resource {
 		},
 
 		SchemaVersion: 1,
-		Schema: AppendBaseAttrSchema(map[string]*schema.Schema{
+		Schema: map[string]*schema.Schema{
 			"attachable_access_entity_profile_dn": {
 				Type:     schema.TypeString,
 				Required: true,
 				ForceNew: true,
 			},
-			"annotation": {
+			"annotation": &schema.Schema{
 				Type:     schema.TypeString,
 				Optional: true,
 				Computed: true,
+				DefaultFunc: func() (interface{}, error) {
+					return "orchestrator:terraform", nil
+				},
```

Comment:
same comment as Akini on other file.

Reply:
Added a GetAnnotation function for uniform schema implementation.